Pod/Demo/AppDelegate.swift

```diff
@@ -47,9 +47,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
       tutorialController.navigationItem.leftBarButtonItem = leftButton
       tutorialController.navigationItem.rightBarButtonItem = rightButton
 
-      addPages()
-      addAnimations()
-      addBackgroundViews()
+      addSlides()
+      addScene()
```

Comment:
What's an scene and what does it do?

Mind adding some explanation on what this PR is about?